src/instructlab/configuration.py

```diff
@@ -29,25 +30,67 @@
 # Local
 from . import log
 
+
+class STORAGE_DIR_NAMES:
+    ILAB = "instructlab"
+    DATASETS = "datasets"
+    CHECKPOINTS = "checkpoints"
+    MODELS = "models"
+    TAXONOMY = "taxonomy"
+    INTERNAL = (
+        "internal"  # for storing all ilab-internal files the user doesn't need to see
+    )
+
+
+def get_home_dir(*subdir: str) -> str:
+    home = os.getenv("HOME")
```

Comment:
```suggestion
    home = str(Path.home())
```

How about this instead of the entire function?

Reply:
Added it to check for both `Path.home()` and `os.getenv("HOME")`